src/java/org/apache/cassandra/cql3/Ordering.java

```diff
@@ -105,7 +105,7 @@ public SingleRestriction toRestriction()
         {
             return new SimpleRestriction(ColumnsExpression.singleColumn(columnMetadata, tableMetadata),
                                          Operator.ANN,
-                                         Terms.of(vectorValue));
+                                         Terms.of(vectorValue), false);
```

Comment:
nit/formatting: I'd bump `false` to the next line so the arguments are all at the same indentation level.